packages/venia-concept/webpack.config.js

```diff
@@ -12,15 +12,15 @@ const {
 } = require('@magento/pwa-buildpack');
 const path = require('path');
 
-const UglifyPlugin = require('uglifyjs-webpack-plugin');
+const TerserPlugin = require('terser-webpack-plugin');
```

Comment:
What's the justification for this change?

Reply:
The [terser homepage](https://github.com/terser-js/terser) explains it pretty well. Webpack 4 tree shaking requires a minifier that understands ES6 natively. Without Terser, this PR would not produce the smaller builds it produces!

Reply:
Yeah, I can see now that they simply gave up on `uglify-es`. 😕